planet/clients/subscriptions.py

```diff
@@ -73,7 +43,22 @@ class SubscriptionsClient:
     """
 
     def __init__(self, session: Session) -> None:
-        self.session = session
+        self._session = session
+
+        self._base_url = BASE_URL
+        if self._base_url.endswith('/'):
+            self._base_url = self._base_url[:-1]
+
+    def _request(self, url, method, data=None, params=None, json=None):
+        return Request(url, method=method, data=data, params=params, json=json)
+
+    async def _do_request(self, request: Request) -> Response:
+        """Submit a request and get response.
+
+        Parameters:
+            request: request to submit
+        """
+        return await self._session.request(request)
```

Comment:
@jreiberkyle in #629 I found that _request and _do_request aren't needed. We can use the bodies of the functions instead.